app/src/main/java/com/example/concordia_campus_guide/models/WalkingPoint.java

```diff
@@ -87,6 +88,22 @@ public Coordinates getCoordinate() {
         return coordinate;
     }
 
+    public double getEuclideanDistanceFrom(Coordinates coordinates){
+        return coordinate.getEuclideanDistanceFrom(coordinates);
+    }
+
+    public LatLng getLatLng(){
+        return coordinate.getLatLng();
+    }
+
+    public double getLatitude(){
+        return coordinate.getLatitude();
+    }
+
+    public double getLongitude(){
+        return coordinate.getLongitude();
+    }
```

Comment:
Do we need both ? why don't we keep only the method that return LatLng ?

Reply:
You need all of these because this ensures that the `LatLng` is not coupled to the class calling `getLongitude` or `getLatitude`